<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PalitrColor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="colorbox">
               <item>
                <widget class="QLabel" name="colorlb_1">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="codelbl_1">
                 <property name="text">
                  <string>code</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="namelbl_1">
                 <property name="text">
                  <string>color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="chosBtn_1">
                 <property name="text">
                  <string>вибрати</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="createBtn_1">
                 <property name="text">
                  <string>створити</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="colorbox_2">
               <item>
                <widget class="QLabel" name="colorlbl_2">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="codelbl_2">
                 <property name="text">
                  <string>code</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="namelbl_2">
                 <property name="text">
                  <string>color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="chosBtn_2">
                 <property name="text">
                  <string>вибрати</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="createBtn_2">
                 <property name="text">
                  <string>створити</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="colorbox_3">
               <item>
                <widget class="QLabel" name="colorlbl_3">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="codelbl_3">
                 <property name="text">
                  <string>code</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="namelbl_3">
                 <property name="text">
                  <string>color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="chosBtn_3">
                 <property name="text">
                  <string>вибрати</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="createBtn_3">
                 <property name="text">
                  <string>створити</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="colorbox_4">
               <item>
                <widget class="QLabel" name="colorlbl_4">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="codelbl_4">
                 <property name="text">
                  <string>code</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="namelbl_4">
                 <property name="text">
                  <string>color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="chosBtn_4">
                 <property name="text">
                  <string>вибрати</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="createBtn_4">
                 <property name="text">
                  <string>створити</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="accountlbl">
             <property name="text">
              <string>акаунт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="regbtn">
             <property name="text">
              <string>увійти</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="palletename">
           <property name="placeholderText">
            <string>назва палітри</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="savebtn">
             <property name="text">
              <string>зберегти</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadbtn">
             <property name="text">
              <string>заванажити</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>220</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>деталі</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>вийти</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
